src/states/node.rs

```diff
@@ -2377,8 +2378,12 @@ impl Node {
         Ok(())
     }
 
+    // Send signed_msg on route. Hop if the name of the peer we received this from, or our name if
```

Comment:
Typo: should be "Hop is the name".

Also in comment lines/blocks, we generally wrap variables in ticks.  This applies to a few places in this PR.